### ISSUE TITLE
Error in widget, "Unsupported operand type(s) for |: 'type' and 'type'" - Possibly Conda related

### ISSUE THREAD

**Issue by TheIronMarx** [davidhi7/ddcci-plasmoid#25]
Python version 3.9.12
ddcutil version 1.4.1

`ddcutil detect` output 
```Display 1
   I2C bus:  /dev/i2c-6
   DRM connector:           card0-DP-1
   EDID synopsis:
      Mfg id:               ACR - Acer Technologies
      Model:                XB271HU
      Product code:         1168  (0x0490)
      Serial number:        #ASNnbwQFBgfd
      Binary serial number: 1663072682 (0x632079aa)
      Manufacture year:     2016,  Week: 32
   VCP version:         2.2

Display 2
   I2C bus:  /dev/i2c-7
   DRM connector:           card0-DP-2
   EDID synopsis:
      Mfg id:               GSM - Goldstar Company Ltd (LG)
      Model:                27GN950
      Product code:         23450  (0x5b9a)
      Serial number:        206NTVS7U671
      Binary serial number: 266671 (0x000411af)
      Manufacture year:     2022,  Week: 6
   VCP version:         2.1
```

The standalone and system tray widgets both display the same error, which is also seen when running `python -m ddcci_plasmoid_backend --debug detect` (excluding initial DEBUG line)
```
DEBUG __main__: Running version 0.1.5 in debug mode
Traceback (most recent call last):
  File "/home/fargonaut/dev/dependencies/anaconda/anaconda3/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/fargonaut/dev/dependencies/anaconda/anaconda3/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/fargonaut/.local/lib/python3.9/site-packages/ddcci_plasmoid_backend/__main__.py", line 97, in <module>
    main()
  File "/home/fargonaut/.local/lib/python3.9/site-packages/ddcci_plasmoid_backend/__main__.py", line 28, in main
    def handle_error(error: str | subprocess.CalledProcessError) -> NoReturn:
TypeError: unsupported operand type(s) for |: 'type' and 'type'
```

I'm worried it's a conda issue, something I honestly don't know much about, but so far hasn't caused issues with other python scripts. I thought I'd run it by here before I dig deeper.

I looked at #12, but I'm not sure there's a parallel solution here. Any thoughts?

**Comment by davidhi7** [comment]
Apparently the `|` operator for type unions was only introduced in Python 3.10. Simply replacing all occurrences of `A | B` with `typing.Union[A, B]` should fix this.

**Comment by davidhi7** [comment]
Forgott to mention but I will not be able to fix this the next couple of weeks. So as a temporary workaround consider upgrading to python 3.10 or later.

**Comment by davidhi7** [comment]
Fixed in release 0.1.6.
